packages/grafana-data/src/events/common.ts

```diff
@@ -0,0 +1,34 @@
+import { DataFrame } from '../types';
+import { BusEventWithPayload } from './types';
+
+/** @alpha */
+export interface DataHoverPayload {
+  raw: any; // Original mouse event (includes pageX etc)
+
+  x: Record<string, any>; // { time: 5678 },
+  y: Record<string, any>; // { __fixed: 123, lengthft: 456 }  // each axis|scale gets a value
+
+  data?: DataFrame; // source data
+  rowIndex?: number; // the hover row
+  columnIndex?: number; // the hover column
+  dataId?: string; // identifying string to correlate data between publishers and subscribers
+  source?: string[]; // source path from where the event originates
```

Comment:
Perhaps this should not be part of the payload. A new BusEventWithSource with the source property would make more sense